<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>1500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BP - Epidemic outbreak </string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsViewDefault">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>215</y>
      <width>450</width>
      <height>260</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>310</width>
      <height>202</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(170, 212, 203)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>338</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(170, 212, 203);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCOUNT">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>214</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во пациентов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>294</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выздоровевших</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>232</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>268</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Здоровых</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRecoveredCount">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>297</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(178, 102, 181)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHealthyCount">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>270</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 143, 116)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>242</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Больных</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSickCount">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>243</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(152, 80, 29)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>257</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>283</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>310</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>153</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>1. Отсутствие мер
предотвращения распространения болезни</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewChart">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>238</y>
      <width>330</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(237, 239, 239);
border: 0px</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewDistance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>598</y>
      <width>450</width>
      <height>260</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>310</width>
      <height>202</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(170, 212, 203)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>534</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>13</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>2. Соблюдение частью
граждан социального дистанцирования</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>694</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>616</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelRecoveredCount_2">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>681</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(178, 102, 181)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCOUNT_2">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>598</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во пациентов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>678</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выздоровевших</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>652</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Здоровых</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSickCount_2">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>627</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(152, 80, 29)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>641</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>626</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Больных</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewChart_Distance">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>622</y>
      <width>330</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(237, 239, 239);
border: 0px</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Distance">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>720</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(170, 212, 203);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHealthyCount_2">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>654</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 143, 116)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>668</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/images/images/Barnaul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 212, 203)</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_4</zorder>
   <zorder>line_3</zorder>
   <zorder>line_2</zorder>
   <zorder>line</zorder>
   <zorder>graphicsViewDefault</zorder>
   <zorder>pushButton</zorder>
   <zorder>LabelCOUNT</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>labelRecoveredCount</zorder>
   <zorder>labelHealthyCount</zorder>
   <zorder>label_3</zorder>
   <zorder>labelSickCount</zorder>
   <zorder>label_4</zorder>
   <zorder>graphicsViewChart</zorder>
   <zorder>graphicsViewDistance</zorder>
   <zorder>label_5</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>labelRecoveredCount_2</zorder>
   <zorder>LabelCOUNT_2</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>labelSickCount_2</zorder>
   <zorder>label_8</zorder>
   <zorder>graphicsViewChart_Distance</zorder>
   <zorder>pushButton_Distance</zorder>
   <zorder>labelHealthyCount_2</zorder>
   <zorder>label_9</zorder>
   <zorder>line_9</zorder>
   <zorder>line_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
